README.md

```diff
@@ -1,92 +1,39 @@
 # Mentorship System (Android)
 
-This is the Android application of the Mentorship System.
+| Branch | [Travis](https://travis-ci.org/) | [Codacy](https://www.codacy.com/) |
+| :---: | :---: | :---: |
+| [master](https://github.com/systers/mentorship-android/tree/master) | [![Build Status](https://travis-ci.org/systers/mentorship-android.svg?branch=master)](https://travis-ci.org/systers/mentorship-android) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/ee27e44e4ac646e0afe440173ea47823?branch=master)](https://www.codacy.com/app/m-murad/mentorship-android) |
+| [develop](https://github.com/systers/mentorship-android/tree/develop) | [![Build Status](https://travis-ci.org/systers/mentorship-android.svg?branch=develop)](https://travis-ci.org/systers/mentorship-android) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/ee27e44e4ac646e0afe440173ea47823?branch=develop)](https://www.codacy.com/app/m-murad/mentorship-android) |
 
-The backend for this project can be found on [systers/mentorship-backend](https://github.com/systers/mentorship-backend) GitHub repository.
+[Mentorship System](https://github.com/systers/mentorship-backend) is an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period.
```

Comment:
Should this link to backend repo or not link to anything at all?

Reply:
Didn't get your question

Reply:
The "Mentorship System" is linking to the backend project. Is this how you really wanted to have this?

Reply:
Yes, you have a better proposition.